### PR TITLE
Add note about the avd loader limited format

### PR DESCRIPTION
**Pull request by thomasponsmicroej**

### CODE REVIEW

File: ApplicationDeveloperGuide/UI/MicroVG/avdLoader.rst [hunk 1]
```diff
@@ -81,15 +81,24 @@ It defines a 100 x 100 image with two paths: the first one with a solid color fi
...
 
-   The image generator tool provides a way to make a Vector Drawable compatible with the library.
+   To make a Vector Drawable compatible with the library, use the image generator tool to convert the AVD into a compatible version.
    See :ref:`this section <section.avdloader.convert>` for more information.
 
 
+Why is the Supported Format a Subset of Android Vector Drawable?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The library only supports a subset of the `Vector Drawable specification <https://developer.android.com/reference/android/graphics/drawable/VectorDrawable>`_.
```
Comment (gbalan-microej) [inline ApplicationDeveloperGuide/UI/MicroVG/avdLoader.rst:95]:
```suggestion
..attention: The library only supports a subset of the `Vector Drawable specification <https://developer.android.com/reference/android/graphics/drawable/VectorDrawable>`_.
```

File: ApplicationDeveloperGuide/UI/MicroVG/avdLoader.rst [hunk 2]
```diff
@@ -81,15 +81,24 @@ It defines a 100 x 100 image with two paths: the first one with a solid color fi
...
 If the input Vector Drawable does not comply with this format, the library will throw an exception.
 
 .. note::
 
-   The image generator tool provides a way to make a Vector Drawable compatible with the library.
+   To make a Vector Drawable compatible with the library, use the image generator tool to convert the AVD into a compatible version.
    See :ref:`this section <section.avdloader.convert>` for more information.
 
 
+Why is the Supported Format a Subset of Android Vector Drawable?
```
Comment (gbalan-microej) [inline ApplicationDeveloperGuide/UI/MicroVG/avdLoader.rst:92]: The spec of the titles is to be shorter. This peace of doc does not respect this rule, even truer with this tile

File: ApplicationDeveloperGuide/UI/MicroVG/avdLoader.rst [hunk 3]
```diff
@@ -81,15 +81,24 @@ It defines a 100 x 100 image with two paths: the first one with a solid color fi
...
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The library only supports a subset of the `Vector Drawable specification <https://developer.android.com/reference/android/graphics/drawable/VectorDrawable>`_.
+
+This is because the AVD Loader is designed to load AVDs at runtime on embedded devices.
```
Comment (gbalan-microej) [inline ApplicationDeveloperGuide/UI/MicroVG/avdLoader.rst:97]:
```suggestion
The AVD Loader is designed to load AVDs at runtime on embedded devices.
```

File: ApplicationDeveloperGuide/UI/MicroVG/avdLoader.rst (hunk 2 above)
Comment (thomasponsmicroej) [reply ApplicationDeveloperGuide/UI/MicroVG/avdLoader.rst]: ok, I proposed shorter titles

File: ApplicationDeveloperGuide/UI/MicroVG/avdLoader.rst (hunk 3 above)
Comment (thomasponsmicroej) [reply ApplicationDeveloperGuide/UI/MicroVG/avdLoader.rst]: applied

**Pull request merged**